src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js

```diff
@@ -1,5 +1,7 @@
 import {normalTextMixin} from './lib.js'
 
+// TODO(matthew): Can we get rid of this????
```

Comment:
Ok, sounds good. May want to revisit this later; not sure we should really support two standards, but not important at the moment. Will remove todo.